chase/frontend/app/login/participant/page.tsx

```diff
@@ -0,0 +1,67 @@
+"use client";
+import React, { useState, useContext } from "react";
+import Image from "next/image";
+
+import UsernameLogin from "../../../components/login/username";
+import PasswordLogin from "../../../components/login/password";
+
+export default function loginVorsitz() {
+  const [loginStage, changeLoginState] = useState(0);
+
+  return (
+    <>
+      <div
+        className="flex justify-stretch h-screen"
+        style={{ backgroundColor: "#eee" }}
+      >
+        <div className="flex-1 flex justify-center">
+          <div
+            className="flex-1 flex flex-col justify-stretch items-center rounded-2xl shadow-md m-10 max-w-lg"
+            style={{ backgroundColor: "#fff" }}
+          >
+            <div className="flex justify-center items-center m-10">
+              <Image
+                src={"/logo/png/chase_logo_blue_text.png"}
+                alt="Logo"
+                width={400}
+                height={128}
+              />
+            </div>
+            {(loginStage === 0 && (
+              <>
+                <div className="flex-1 flex flex-col justify-center items-center">
+                  <UsernameLogin changeLoginState={changeLoginState} />
+                  <p className="mt-10 text-gray-400 hover:text-black underline text-xs text-center">
+                    <a href="/login/chair">Stattdessen als Vorsitz anmelden</a>
```

Comment:
Unsure if we can use the native anchor tags. Either please research if next intercepts native anchor tags or use the link component